camping-omnibus.gemspec-old

```diff
@@ -2,10 +2,10 @@
 # camping_omni
 
 NAME = "camping"
-BRANCH = "2.3"
+BRANCH = "3.0.0"
 GIT = ENV['GIT'] || "git"
 REV = `#{GIT} rev-list HEAD`.strip.split.length
-VERS = ENV['VERSION'] || (REV.zero? ? BRANCH : [BRANCH, REV] * '.')
+VERS = ENV['VERSION'] || BRANCH # (REV.zero? ? BRANCH : [BRANCH, REV] * '.')
```

Comment:
I don't follow what's happening here, were you going for string interpolation with `BRANCH #` ?

Reply:
This file isn't really used at the moment, but generally the version number has a major and a commit number, so 2.532, for version w, and 532 total commits. We're moving to Semantic versioning so this isn't really necessary at the moment.

Camping Omnibus is something that we'll need to discuss at length soon, to figure out exactly what to do with it in the future.